### ISSUE TITLE
Values not removed from destination JDBC Database

### ISSUE THREAD

**Issue by MSSputnik** [lsc-project/lsc#80]
I have a jdbcDatabase Source and a jdbcDatabase Destination.
Everything is working as expected except values get not removed from destination when removed from source.
The log shows correctly:
```
Jun 15 14:47:57 - INFO  - # Updating object 90000011CHJCL72421000036 for MySyncTask
# Sat Jun 15 14:47:57 CEST 2019
dn: 90000011CHJCL72421000036
changetype: modify
delete: lastname_2
-
```

But in the destination, the original value remains. No deletion of the value happens.

I tracked it down to the Class org.lsc.service.AbstractJdbcService
Functions 
- public static Map<String, Object> getAttributesMap(List<LscDatasetModification> lscAttributeModifications)
- public static Map<String, String> getAttributesMap(LscDatasets lscAttributes)

The else statement is missing, in `if(lam.getValues().size() > 0) {`
I added:
```
} else {
				values.put(lam.getAttributeName(), null);
```
and for the 2nd function
```
} else {
				values.put(entry.getKey(), null);
```

With this change, the values are correctly removed from my destination Database.

**Comment by coudot** [comment]
Hello @MSSputnik 

thanks for this issue, do you think you could create a pull request? The best would be to add a unit test to prove the bug has been fixed.

**Comment by MSSputnik** [comment]
Hi,
I will try my best to add the unit test.
Martin

**Comment by MSSputnik** [comment]
I just created the Pull request.

**Comment by soisik** [comment]
Fixed with PR #81
